Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java

```diff
@@ -51,21 +51,52 @@
         public String username;
         public String avatarUrl;
 
+        /**
+         * calls {@link CommandResult#CommandResult(String, List, boolean)} with default arguments.
+         * @see CommandResult#CommandResult(String, List, boolean)
+         */
         public CommandResult() { this(null); }
+
+        /**
+         * calls {@link CommandResult#CommandResult(String, List, boolean)} with default arguments.
+         * @param content Output message content
+         * @see CommandResult#CommandResult(String, List, boolean)
+         */
         public CommandResult(String content) {
             this(content, null, true);
         }
+
+        /**
+         * @param content Output message content
+         * @param embeds Embeds to include in the command output. Requires <code>send</code> to be false.
```

Comment:
is this really true? has someone tried what happens if u try to sned embeds? xD

Reply:
it is locked in aliucord api

Reply:
oh okay thats good